inst/eDISH_app/ui.R

```diff
@@ -0,0 +1,38 @@
+library(shiny)
+
+navbarPage("eDISH Shiny app", id = "inTabset",
+           tabPanel(title = "Data", value = "data",
+                    fluidRow(
+                      column(3,
+                             wellPanel(
+                               h3("Data upload"), 
+                               fileInput("datafile", "Upload a csv or sas7bdat file",accept = c(".sas7bdat", ".csv"), multiple = TRUE),
+                              radioButtons("select_file","Select file for eDISH chart", choices = "No files available")
+                             )
+                      ),
+                      column(6, 
+                             fluidRow(
+                               wellPanel( 
+                               h3("Data preview"), 
+                               div(DT::dataTableOutput("data_preview"), style = "font-size: 75%")
+                             )
+                             ),
+                             fluidRow(
+                               br(),
+                               tags$style(type='text/css', '#detectStandard_msg {font-size:23px;}'),
+                               uiOutput("detectStandard_msg"),
+                               actionButton("view_chart","View Chart")
```

Comment:
If "No Standard found", perhaps one would show "configure settings" to point to the settings tab in the button below, instead of pointing to the chart tab?

Also if ADaM or STDM is detected, perhaps tell users explicitly that they probably can skip the setting tab.